python/cuml/svm/svr.pyx

```diff
@@ -294,21 +281,14 @@ class SVR(SVMBase, RegressorMixin):
 
         return self
 
+    @generate_docstring(return_values={'name': 'preds',
```

Comment:
maybe return values name should default to preds for predict or fit_predict methods?

Reply:
I think it is a great idea, but I was thinking this could be a future improvement to avoid increasing the size/complexity of the PR, what do you think?